pmm-app/src/pmm-qan/panel/components/Details/Example/Example.tools.tsx

```diff
@@ -1,11 +1,11 @@
 import Highlight from 'react-highlight.js';
 import React from 'react';
 import sqlFormatter from 'sql-formatter';
-import { DATABASE } from '../Details.constants';
 import { ReactJSON } from '../../../../../shared/components/Elements/ReactJSON/ReactJSON';
+import { Databases } from '../Details.types';
```

Comment:
Can we make the import path shorter on line 4: `shared/components/Elements/ReactJSON`?